demo/src/app/components/+datepicker/demos/outside-click/outside-click.html

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 col-md-4 form-group">
+    <input class="form-control" placeholder="Datepicker" bsDatepicker [outsideClick]="true">
```

Comment:
Add some text that will let a user know what differences these two datepickers have.